import_processor/readme/USAGE.rst

```diff
@@ -0,0 +1,31 @@
+This module provides a set of pre-defined functions and wizards to facilitate importing data into Odoo.
+
+The module offers predefined variables that can be used when writing code snippets in the view.
+
+Note: Only pre-defined variables can be used in code snippets.
+
+The module consists of three main processes:
+
+1. Pre-Processor:
+   - At this stage, you can define code snippets that will be executed before importing the records in the view.
+
+2. Processor:
+   - At this stage, you can define code snippets that will be executed during the import process.
+
+3. Post-Processor:
+   - At this stage, you can define code snippets that will be executed after the records have been imported.
+
+Use Cases
+~~~~~~~~~
+
+1. Navigate to Settings > Technical > Import Processors and create a new Import Processor for CSV files with the following settings:
+   - Name: "CSV"
+   - File Type: CSV
+   - User can also define Chunk size if necessary
+   - Write the necessary code snippets in the Pre-Processor, Processor, and Post-Processor pages at the bottom of the form view.
+
+2. Save the record and go to Contacts > Favorites > Import Processor.
```

Comment:
Technically wrong because Contacts is only the case if you define an import processor for res.partner. It would be better to write it like this: ```Go to a view with the model of your configured model and click on `Favorites > Import Processor` to open the wizard and start the import```